README.md

```diff
@@ -2,58 +2,57 @@
 
 Telegram bot for chats administration.
 
+[![Build Status](https://dev.azure.com/GrinderDotNetRu/Grinder/_apis/build/status/Keroosha.grinder?branchName=master)](https://dev.azure.com/GrinderDotNetRu/Grinder/_build/latest?definitionId=1&branchName=master)
+
 ## Configuration and build
 
-To build run you have to add a `bot_config.json` file on path `src\Grinder\`
+Just open project with Rider/VS 2019 and run build
 
-File structure:
+## Running
 
-```
-{
-  "Token": ...,
-  "ChannelId": ...,
-  "AdminUserId": ...,
-  "Socks5Proxy": {
-    "Hostname": "...",
-    "Port": ...,
-    "Username": "...",
-    "Password": "..."
-  },
-  "ChatsToMonitor": [
-    "chat @username",
-    ...
-  ],
-  "AllowedUsers": [
-    "username without at sign",
-    ...
-  ]
-}
-```
+### Using a Docker container
 
-`Token` - bot api token.
+You can just spawn new container with bot from: ***keroosha/grinder*** repo
 
-`ChannelId` - telegram channel id where bot whill write command logs.
+**(DO NOT FORGET ABOUT ENVIRONMENT VARIABLES)**
 
-`AdminUserId` - telegram user id who will be able to send private messages to bot.
+```bash
+docker run -d keroosha/grinder
+```
 
-`Socks5Proxy` - Socks5 proxy configuration.
+Or you can use already configured ***docker-compose.yml*** but, some action required before starting
 
-`ChatsToMonitor` - chats where bot will read messages and replyes.
+1. Create a ***config.env*** file with environments
+2. Change data path in ***docker-compose.yml***
+3. Setup database
```

Comment:
How am I supposed to do that? Not a very descriptive guidance I'd say.

Reply:
Database section require @Liminiens

Anyting else I tried to make more descriptive